configure.ac

```diff
@@ -148,7 +152,7 @@ AC_ARG_ENABLE(xmp,
     AC_HELP_STRING([--disable-xmp],
                    [build without xmp support]))
 msg_xmp=no
-if test "x$enable_xmp" != "xno"; then
+AS_IF([test "x$enable_xmp" != "xno"],[
```

Comment:
This AS_IF change is not necessarily required by this PR. It founds its way into the patch I ship with the Debian package. But it is off-topic here. Sorry for the confusion.